controller/src/main/java/org/jboss/as/controller/OperationContextImpl.java

```diff
@@ -846,7 +846,7 @@ private void ensureWriteLockForRuntime() {
                 ExecutionStatus origStatus = executionStatus;
                 try {
                     executionStatus = ExecutionStatus.AWAITING_STABILITY;
-                    modelController.awaitContainerStability(timeout, TimeUnit.MILLISECONDS, false);
+                    modelController.awaitContainerStability(timeout, TimeUnit.MILLISECONDS, false); // interruption forbidden - see WFCORE-6158
```

Comment:
I guess you wanted to use see WFCORE-6157, instead of seeing WFCORE-6158. We can correct it after merging, but could you confirm @ropalka ?